colors/nord.vim

```diff
@@ -601,6 +601,49 @@ hi! link NERDTreeHelp Comment
 hi! link CtrlPMatch Keyword
 hi! link CtrlPBufferHid Normal
 
+" vim-clap
+" > liuchengxu/vim-clap
+call s:hi("ClapInput", s:nord4_gui, s:nord2_gui, s:nord3_term, s:nord1_term, "", "")
+call s:hi("ClapDisplay", s:nord5_gui, s:nord1_gui, s:nord5_term, s:nord1_term, "", "")
+call s:hi("ClapPreview", s:nord4_gui, s:nord2_gui, s:nord3_term, s:nord1_term, "", "")
+call s:hi("ClapSelected", s:nord7_gui, "", s:nord7_term, "", s:underline, "")
+call s:hi("ClapCurrentSelection", s:nord7_gui, "", s:nord8_term, "", s:bold, "")
+call s:hi("ClapNoMatchesFound", s:nord13_gui, "", s:nord13_term, "", "", "")
+let clap_matches = [
+        \ [s:nord11_gui, s:nord11_term] ,
+        \ [s:nord13_gui, s:nord13_term] ,
+        \ [s:nord14_gui, s:nord14_term] ,
+        \ [s:nord11_gui, s:nord11_term] ,
+        \ [s:nord13_gui, s:nord13_term] ,
+        \ [s:nord14_gui, s:nord14_term] ,
+        \ [s:nord11_gui, s:nord11_term] ,
+        \ [s:nord13_gui, s:nord13_term] ,
+        \ ]
+let idx = 1
+for g in clap_matches
+  call s:hi("ClapMatches" . idx, g[0], "", g[1], "", "", "")
+  let idx += 1
+endfor
+let clap_fuzzy_matches = [
+        \ [s:nord11_gui, s:nord11_term] ,
+        \ [s:nord13_gui, s:nord13_term] ,
+        \ [s:nord14_gui, s:nord14_term] ,
+        \ [s:nord11_gui, s:nord11_term] ,
+        \ [s:nord13_gui, s:nord13_term] ,
+        \ [s:nord14_gui, s:nord14_term] ,
+        \ [s:nord11_gui, s:nord11_term] ,
+        \ [s:nord13_gui, s:nord13_term] ,
+        \ [s:nord14_gui, s:nord14_term] ,
+        \ [s:nord11_gui, s:nord11_term] ,
+        \ [s:nord13_gui, s:nord13_term] ,
+        \ [s:nord14_gui, s:nord14_term] ,
+        \ ]
+let idx = 1
+for g in clap_fuzzy_matches
+  call s:hi("ClapFuzzyMatches" . idx, g[0], "", g[1], "", "", "")
+  let idx += 1
+endfor
```

Comment:
Good work! I'm not maintainer of this repo, but may I suggest a slight improvement over this code?

It seems `clap_matches` and `clap_fuzzy_matches` are pretty much the same (the only difference is length). In fact, they are only 3 colors that we want repeat over and over again. So my suggestion is simply maintain a table of 3 colors, and pick the right one based on the index of `ClapMatch` we're creating.

The only side effect is that we will produce more highlight rules for `ClapMatch` than the plugin supports (12 instead of 8), but in my opinion it's not a big deal. What do you think?


```suggestion
let clap_matches = [
        \ [s:nord11_gui, s:nord11_term] ,
        \ [s:nord13_gui, s:nord13_term] ,
        \ [s:nord14_gui, s:nord14_term] ,
        \ ]
let idx = 1
while idx <= 12
  let clap_match_color = clap_matches[idx % len(clap_matches) - 1]
  call s:hi("ClapMatches" . idx, clap_match_color[0], "", clap_match_color[1], "", "", "")
  call s:hi("ClapFuzzyMatches" . idx, clap_match_color[0], "", clap_match_color[1], "", "", "")
  let idx += 1
endfor
````

Reply:
LGTM @arcticicestudio can decide